README.md

```diff
@@ -228,6 +228,8 @@ This project uses some modern Android libraries and source codes.
 * [Injected ViewModel Provider](https://github.com/evant/injectedvmprovider) (evant)
 * [Google I/O 2018](https://github.com/google/iosched) (Google)
 * [TimetableLayout](https://github.com/MoyuruAizawa/TimetableLayout) (MoyuruAizawa)
+* [Android Architecture Components samples](https://github.com/android/architecture-components-samples) (Google)
+  * [AutoClearedValue](https://github.com/android/architecture-components-samples/blob/master/GithubBrowserSample/app/src/main/java/com/android/example/github/util/AutoClearedValue.kt)
```

Comment:
Can you use link specific commit? This link uses master. When the file removed we can not access the page.
You can access URL when pressing `y` on the page.
https://github.com/android/architecture-components-samples/blob/master/GithubBrowserSample/app/src/main/java/com/android/example/github/util/AutoClearedValue.kt

Reply:
I fixed it to specified commit used in [this app](https://github.com/DroidKaigi/conference-app-2020/blob/5dbba2f359a713a8e5181625d6ffbdc4598171d9/corecomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2020/util/AutoClearedValue.kt).

And the tips of shortcut (`y`) is help for me. Thanks for review!